### DISCUSSION TITLE
Question: Cert Manager needed?

### DISCUSSION THREAD

**Discussion by lukonjun** [actions/actions-runner-controller#1347] [Questions]
Hello, just trying out the actions-runner-controller. I was just curious why exactly do we need cert manager, what would not work without it?

**Comment by toast-gear** [comment]
> I was just curious why exactly do we need cert manager

To manage the admission webhook certificate which is not optional. If you don't want the cert-manager dependency which not every does you can roll your own cert https://github.com/actions-runner-controller/actions-runner-controller#using-without-cert-manager

> what would not work without it?

The entire solution

**Comment by mumoshu** [comment]
I think this should better moved to Discussions :)
